api/namex/models/word_classification.py

```diff
@@ -0,0 +1,49 @@
+""""word classification classifies all words in a name approved by an exmainer to be used for auto-approval
+
+"""
+from . import db, ma
+from datetime import datetime
+from sqlalchemy.orm import backref
+
+
+class WordClassification(db.Model):
+    __tablename__ = 'word_classification'
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    classification = db.Column('word_classification',db.String(4),default='NONE',nullable=False,index=True)
+    word = db.Column('word', db.String(1024), nullable=False, index=True)
+    lastNameUsed = db.Column('last_name_used',db.String(1024))
+    lastPrepName = db.Column('last_prep_name',db.String(1024))
+    frequency = db.Column('frequency', db.BIGINT)
+    approvedBy = db.Column('approved_by', db.Integer, db.ForeignKey('users.id'))
+    approvedDate = db.Column('approved_dt', db.DateTime(timezone=True), default=datetime.utcnow)
+    startDate = db.Column('start_dt', db.DateTime(timezone=True), default=datetime.utcnow)
+    endDate = db.Column('end_dt', db.DateTime(timezone=True), default=datetime.utcnow)
+    lastUpdatedBy = db.Column('last_updated_by', db.Integer, db.ForeignKey('users.id'))
+    lastUpdateDate =  db.Column('last_update_dt', db.DateTime(timezone=True), default=datetime.utcnow)

```

Comment:
You might want to add an 'onupdate' value so that this updates automatically every time the record is changed (if that's the functionality you intended. Here's the namex request version:
```
    lastUpdate = db.Column('last_update', db.DateTime(timezone=True), default=datetime.utcnow, onupdate=datetime.utcnow)
```

Reply:
I will do that.  Thanks.